crates/enarx-config/src/lib.rs

```diff
@@ -13,6 +13,41 @@ use std::{collections::HashMap, ops::Deref};
 use serde::{de::Error as _, Deserialize, Deserializer};
 use url::Url;
 
+/// Configuration file template
+pub const CONFIG_TEMPLATE: &str = r#"# Configuration for a WASI application in an Enarx Keep
+
+# Environment variables
+# env = { "VAR1" = "var1", "VAR2" = "var2" }
+
+# Arguments
+# args = [ "--argument1", "--argument2=foo" ]
```

Comment:
Again I'd vote for splitting this into multiple lines

Reply:
done